pkgs/by-name/co/code-cursor/package.nix

```diff
@@ -4,96 +4,102 @@
   fetchurl,
   appimageTools,
   makeWrapper,
-  writeScript,
+  undmg,
+
 }:
+
 let
   pname = "cursor";
-  version = "0.44.11";
+  version = "0.45.7";
+
+  inherit (stdenvNoCC) hostPlatform;
 
   sources = {
     x86_64-linux = fetchurl {
-      url = "https://download.todesktop.com/230313mzl4w4u92/cursor-0.44.11-build-250103fqxdt5u9z-x86_64.AppImage";
-      hash = "sha256-eOZuofnpED9F6wic0S9m933Tb7Gq7cb/v0kRDltvFVg=";
+      url = "https://download.todesktop.com/230313mzl4w4u92/cursor-0.45.7-build-250130nr6eorv84-x86_64.AppImage";
```

Comment:
I don't love that the version is just hardcoded into this URL, but given that other parts of the URL will need to be updated for every version and that it's handled by the update script, this looks fine to me.

Reply:
I wish they'd evict the build number and regularize the name across file formats too.